clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionServiceTest.java

```diff
@@ -546,7 +546,7 @@ public void testDropStoragePartitionGracefully() throws NoSuchFieldException, Il
     storageServiceField.setAccessible(true);
     storageServiceField.set(storeIngestionTask, storageService);
 
-    Field storeConfigField = StoreIngestionTask.class.getDeclaredField("storeConfig");
+    Field storeConfigField = StoreIngestionTask.class.getDeclaredField("storeVersionConfig");
```

Comment:
no need to change, but I feel like it can be changed to default access instead of getDefaultField, but doesn't block this change though.